build/pipelines/azure-pipelines.release.yaml

```diff
@@ -4,13 +4,6 @@
 # Store and the Windows image. This pipeline relies on Microsoft-internal resources to run.
 #
 
-schedules:
-- cron: "0 7 * * *"
-  displayName: Daily midnight build
-  branches:
-    include:
-    - master
-
```

Comment:
Stop the schedules, only trigger release pipeline manually after ci-internal passed.

Reply:
It seems to be not necessary to remove this schedule. This schedule is for the master branch, and we should check the ci result before merge any PR into master.

Reply:
If having the schedule here, there is chance that maybe release pipeline starts before ci-internal done.